studygroups/tests/api/test_learning_circle_api.py

```diff
@@ -456,7 +466,8 @@ def test_get_learning_circles_unicode(self):
             lc = StudyGroup.objects.all().last()
             self.assertEqual(resp.json(), {
                 "status": "created",
-                "url": "{}/en/signup/%D8%A7%D9%84%D8%B5%D8%AD%D8%A9-%D8%A7%D9%84%D9%86%D9%81%D8%B3%D9%8A%D8%A9-%D9%84%D9%84%D8%B7%D9%81%D9%84-{}/".format(settings.DOMAIN, lc.pk)
+                "signup_url": "{}://{}/en/signup/%D8%A7%D9%84%D8%B5%D8%AD%D8%A9-%D8%A7%D9%84%D9%86%D9%81%D8%B3%D9%8A%D8%A9-%D9%84%D9%84%D8%B7%D9%81%D9%84-{}/".format(settings.PROTOCOL, settings.DOMAIN, lc.pk),
```

Comment:
@s-kennedy Are we using the signup_url from the API anywhere? I kind of suspect I just put it in because i thought it made sense? But it probably makes fore sense to just return the studygroup_url, since the only person getting this back from the API would be the facilitator who created the learning circle.

Reply:
no we're not, I removed it.